### PR TITLE
added the windows files

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

File: example/samples/install_windows_payload_full.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        {
+          "device": "Ethernet",
+          "ipv4": {
+            "ipAddr": "172.31.128.222",
+            "gateway": "192.168.1.1",
+            "netmask": "255.255.255.0"
+          }
+        }
+      ],
+      "productkey": "YGQ3N-VWHFV-WR2HV-CY2F2-8W98M",
```
Comment (DavidjohnBlodgett) [inline example/samples/install_windows_payload_full.json]: @ tannoa2 is this a valid key? are we allowed to distribute this?

**Comment by tannoa2** [comment]
@davidjohnblodgett all set.  Thanks

**Comment by tannoa2** [comment]
I updated my branch, however since the PR is closed I have to email to gihub to delete the PR.
I sent them an email.

**Comment by tannoa2** [comment]
@DavidjohnBlodgett 
I updated my branch, however since the PR is closed I have to email  gihub to delete the PR.
I sent them an email.
http://stackoverflow.com/questions/18318097/delete-a-closed-pull-request-from-github

**Pull request closed without merging**